### PR TITLE
Add Windows cvar list

### PR DESCRIPTION
**Pull request by Leo40Git**
Current `cvarlist.md` -> `cvarlist_linux.md`
New Windows cvar list is `cvarlist_win.md`.

### CODE REVIEW

File: docs/tf2/cvarlist_win.md [hunk 1]
```diff
@@ -0,0 +1,3558 @@
...
+ds_enable                                : 0        : , "a", "norecord", "cl" : Demo support - enable automatic .dem file recording and features. 0 - Manual, 1 - Auto-record competitive matches, 2 - Auto-record all matches, 3 - Auto-record tournament (mp_tournament) matches
+ds_kill_delay                            : 15       : , "a", "norecord", "cl" : Demo support - maximum time between kills for tracking kill streaks.
+ds_log                                   : 1        : , "a", "norecord", "cl" : Demo support - log kill streak and bookmark events to an associated .txt file.
+ds_mark                                  : cmd      :                  : Demo support - bookmark (with optional single-word description) the current tick count for the demo being recorded.
+ds_min_streak                            : 4        : , "a", "norecord", "cl" : Demo support - minimum kill streak count before being recorded.
+ds_notify                                : 0        : , "a", "norecord", "cl" : Demo support - text output wh
```
Comment (mastercoms) [inline docs/tf2/cvarlist_win.md]: \`\`\` should be on a new line

**Review by mastercoms — CHANGES_REQUESTED** [review]
Also make sure to add a navigation entry to `mkdocs.yml`.

**Review by mastercoms — APPROVED** [review]

**Comment by mastercoms** [comment]
Thank you for your contribution to mastercomfig!

**Comment by mastercoms** [comment]
@Leo40Git Would it be possible for you to do a similar contribution for launch options? Just run the file listed [here](https://github.com/mastercoms/mastercomfig/tree/release/docs/tf2#launch_options), and it'll let you select the Team Fortress 2 folder for scanning.

**Comment by Leo40Git** [comment]
Sure, I can do that. Just gotta wait for GetLaunchOptions to get itself situated (...is it supposed to take a long time?

**Comment by mastercoms** [comment]
@Leo40Git Maybe you can instead select the bin folder and the tf/bin folder? And then isolate hl2.exe.

**Comment by Leo40Git** [comment]
~~actually it's stuck on "RES2BATCH: extracting strings2.ex_"~~

done, #93

**Pull request merged**